### PR TITLE
Added

### PR DESCRIPTION
**Pull request by deepgandhecha7**
Assg 1 Completed

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+            display: block;
+            margin-left: auto;
+            margin-right: auto;
+        }
+    </style>
+</head>
+<body>
+    <div id="container">
+    <!-- Start your code from here -->
+    <div class="left-side">
```
Comment (ygoyal25) [inline src/index.html:56]: Use proper indentation, the structure can be improved, can be done with very few lines of css

Comment (deepgandhecha7) [reply src/index.html:56]: ok i'll do it

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            <li>List item 3</li>
+            <ul>
+                <li>Child List1</li>
+                <li>Child List2</li>
+                <li>Child List3</li>
+            </ul>
+        </ul>
+    </div>
+    <div class="image_div">
+        <img src="https://www.w3schools.com/html/img_chania.jpg" class="center"/>
```
Comment (DyapaHarishwarReddy) [inline src/index.html:69]: Images are in same column

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    </div>
+    <div class="second_img">
+        <img src="https://www.w3schools.com/html/pic_trulli.jpg" class="center"/>
+    </div>
+    <div class="heading">
```
Comment (DyapaHarishwarReddy) [inline src/index.html:74]: Divisons need to be made

**Review by DyapaHarishwarReddy — COMMENTED** [review]
Images are in the same column divisons need to be corrected

**Pull request merged**